src/test/java/com/puppycrawl/tools/checkstyle/internal/TestUtils.java

```diff
@@ -64,6 +66,31 @@ public static void assertUtilsClassHasPrivateConstructor(final Class<?> utilClas
     }
 
     /**
+     * Checks if stateful field is cleared during check#beginTree.
```

Comment:
`check#beginTree` => `{@link AbstractCheck#beginTree} in check`
Sound good?

Reply:
fixed